src/AuthenticatedAPIClient/tests/AuthenticatedAPIClient.test.jsx

```diff
@@ -313,25 +244,36 @@ describe('AuthenticatedAPIClient request headers', () => {
 describe('AuthenticatedAPIClient ensureValidJWTCookie request interceptor', () => {
   beforeEach(() => {
     PubSub.clearAllSubscriptions();
+    mockAccessToken.value = undefined;
+    mockAccessToken.isExpired = true;
+    mockAccessToken.refresh.mockReset();
   });
 
-  [
-    /*
-    isAuthUrl, mockAccessToken, isAccessTokenExpired, queueRequests,
-    rejectRefreshAccessToken, expects
-    */
-    [true, null, true, false, false, expectRefreshAccessTokenToNotHaveBeenCalled],
-    [true, {}, true, false, false, expectRefreshAccessTokenToNotHaveBeenCalled],
-    [true, null, false, false, false, expectRefreshAccessTokenToNotHaveBeenCalled],
-    [true, {}, false, false, false, expectRefreshAccessTokenToNotHaveBeenCalled],
-    [false, null, true, false, false, expectRefreshAccessTokenToHaveBeenCalled],
-    [false, {}, true, false, false, expectRefreshAccessTokenToHaveBeenCalled],
-    [false, {}, true, false, true, expectRefreshAccessTokenToHaveBeenCalled],
-    [false, {}, true, true, false, expectRefreshAccessTokenToNotHaveBeenCalled],
-    [false, null, false, false, false, expectRefreshAccessTokenToNotHaveBeenCalled],
-    [false, {}, false, false, false, expectRefreshAccessTokenToNotHaveBeenCalled],
-  ].forEach((mockValues) => {
-    testJwtCookieInterceptorFulfillment(...mockValues);
```

Comment:
Many of the cases above are no longer relevant. Some tested the old queuing functionality which is now tested inside AccessToken. The ones that are still relevant are reproduced in an un-dry way below.